server/src/main/java/org/elasticsearch/index/mapper/LegacyGeoShapeFieldMapper.java

```diff
@@ -420,12 +420,15 @@ public PrefixTreeStrategy resolvePrefixTreeStrategy(String strategyName) {
         }
     }
 
+    private final Version indexCreatedVersion;
+
     public LegacyGeoShapeFieldMapper(String simpleName, FieldType fieldType, MappedFieldType mappedFieldType,
                                      Explicit<Boolean> ignoreMalformed, Explicit<Boolean> coerce, Explicit<Orientation> orientation,
                                      Explicit<Boolean> ignoreZValue, Settings indexSettings,
                                      MultiFields multiFields, CopyTo copyTo) {
-        super(simpleName, fieldType, mappedFieldType, ignoreMalformed, coerce, ignoreZValue, orientation, indexSettings,
+        super(simpleName, fieldType, mappedFieldType, ignoreMalformed, coerce, ignoreZValue, orientation,
             multiFields, copyTo);
+        this.indexCreatedVersion = Version.indexCreated(indexSettings);
```

Comment:
I thought I'd never see a point where we use the settings at this stage in the PR ;-)